tox.ini

```diff
@@ -63,7 +63,7 @@ deps =
     ansible
     jmespath
 commands =
-    bash -e -c 'if [ -x lsr_roles2collections.sh ]; then COLLECTION_SRC_PATH=/var/tmp/src COLLECTION_DEST_PATH=/var/tmp/collections ROLES="certificate kdump kernel_settings logging metrics nbde_client storage timesync tlog tuned" {toxinidir}/lsr_roles2collections.sh; cd /var/tmp/collections/ansible_collections/fedora/system_roles/tests; ANSIBLE_COLLECTIONS_PATHS=/var/tmp/collections:/usr/share/ansible/collections ansible-playbook --become --become-user root -e 'ansible_python_interpreter=/usr/bin/python3' -vvvv tests_default.yml; rm -rf /var/tmp/src /var/tmp/collections; fi'
+    bash -e -c 'if [ -x lsr_roles2collections.sh ]; then COLLECTION_SRC_PATH=/var/tmp/src COLLECTION_DEST_PATH=/var/tmp/collections ROLES="certificate kdump kernel_settings logging metrics nbde_client timesync tlog tuned" {toxinidir}/lsr_roles2collections.sh; cd /var/tmp/collections/ansible_collections/fedora/system_roles/tests; ANSIBLE_COLLECTIONS_PATHS=/var/tmp/collections:/usr/share/ansible/collections ansible-playbook --become --become-user root -e 'ansible_python_interpreter=/usr/bin/python3' -vvvv tests_default.yml; rm -rf /var/tmp/src /var/tmp/collections; fi'
```

Comment:
What is the problem here?

Reply:
I guess this is a well known fact (although I did not know...); Fedora/RHEL/CentOS is not running on the travis machine, but something else (debian?) which runs `apt` for installing packages. Thus, roles which require redhat specific packages or contain include_var fallbacks with no default.yml have no luck to pass the basic test...
```
ASK [fedora.system_roles.tlog : install session recording packages] ***********
task path: /var/tmp/collections/ansible_collections/fedora/system_roles/roles/tlog/tasks/main.yml:2
Running apt
<<snip>>
```